### PR TITLE
Update Crusher Docs and Scripts

### PR DESCRIPTION
**Pull request by n01r**
The updated Crusher scripts and documentation were kept very close to the ones for Frontier since they were found to work very well, currently. The original Frontier docs were authored by Axel Huebl <axelhuebl@lbl.gov>.

### CODE REVIEW

File: Docs/source/install/hpc/crusher.rst [hunk 1]
```diff
@@ -17,79 +18,146 @@ If you are new to this system, **please see the following resources**:
...
-  * ``$MEMBERWORK/$proj/``: single user, purged every 90 days (usually smaller quota)
-  * ``$WORLDWORK/$proj/``: shared with all users, purged every 90 days
-  * Note that the ``$HOME`` directory is mounted as read-only on compute nodes.
-    That means you cannot run in your ``$HOME``.
+  * ``$HOME``: per-user directory, use only for inputs, source and scripts; backed up; mounted as read-only on compute nodes, that means you cannot run in it (50 GB quota)
+  * ``$PROJWORK/$proj/``: shared with all members of a project, purged every 90 days, Lustre (recommended)
+  * ``$MEMBERWORK/$proj/``: single user, purged every 90 days, Lustre (usually smaller quota, 50TB default quota)
+  * ``$WORLDWORK/$proj/``: shared with all users, purged every 90 days, Lustre (50TB default quota)
 
+Note: the Orion Lustre filesystem on Frontier and Crusher, and the older Alpine GPFS filesystem on Summit are not mounted on each others machines.
```
Comment (ax3l) [inline Docs/source/install/hpc/crusher.rst:26]: Oh, the FS changed and is primarily Orion now for Crusher, too?

File: Docs/source/install/hpc/crusher.rst [hunk 2]
```diff
@@ -144,6 +212,44 @@ Known System Issues
 
    January, 2023 (OLCFDEV-1284, AMD Ticket: ORNLA-130):
    We discovered a regression in AMD ROCm, leading to 2x slower current deposition (and other slowdowns) in ROCm 5.3 and 5.4.
-   Reported to AMD and fixed for the 5.5 release of ROCm.
 
-   Upgrade ROCm or stay with the ROCm 5.2 module to avoid.
+   June, 2023:
```
Comment (ax3l) [inline Docs/source/install/hpc/crusher.rst]: Shall we just cross-link to :ref:`known Frontier issues <known-crusher-issues>` since they are so similar? :)

**Review by ax3l — APPROVED** [review]
Really appreciate this, @n01r , thank you!

Small suggestions inline :)

**Comment by ax3l** [comment]
Want to add the `cupy` line as well? #4247 :)

File: Docs/source/install/hpc/crusher.rst (hunk 1 above)
Comment (n01r) [reply Docs/source/install/hpc/crusher.rst:26]: Yep, indeed :)
E.g., `$PROJWORK` now leads to `/lustre/orion/proj-shared`.

**Comment by n01r** [comment]
> Want to add the `cupy` line as well? #4247 :)

@ax3l, I would skip this for now if that's okay since the installation that works perfectly on Frontier does not work like that on Crusher, yet.

**Review by ax3l — APPROVED** [review]
Thank you! :)

**Pull request merged**